docs/monitoring.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: default
+title: Monitoring for Migrations
+nav_order: 13
+permalink: /monitoring
+---
+
+
+# Monitoring for Migrations
+{: .no_toc }
+
+Spanner Migration Monitoring for Sharded and Non-Sharded Migrations is now available. The monitoring dahsboard can be accessed through both UI and CLI. Spanner Migration Tool uses the [**Google Cloud Monitoring Service**](https://cloud.google.com/monitoring) to create custom dashboard for each migration.
+
```

Comment:
Start by defining what the migration dashboard is for, what is its utility, what values does it add.

Some excerpts - 

"Allow to different GCP resources involved in a forward minimal downtime migration..."

"Helps understand what is happening in each part of the migration pipeline to provide greater visibility into the health and progress..."

Use something like this as a guide/reference to write out this documentation. Pay attention to the structure and the flow of information. 

https://cloud.google.com/spanner/docs/monitoring-console